commit 6efd6cd46bf2257e674be4933a034542d80944eb
Author: Florian Weimer <fweimer@redhat.com>
Date:   Fri Sep 6 14:07:00 2024 +0200

    manual: Safety annotations for clock_gettime, clock_getres
    
    The annotations are preliminary, for consistency with existing
    annotations on gettimeofday etc.
    
    Reviewed-by: Carlos O'Donell <carlos@redhat.com>

Conflicts:
	manual/time.texi (fixing typo in context)

diff --git a/manual/time.texi b/manual/time.texi
index 0c7a942b4c16946a..404d124d1bfdca5c 100644
--- a/manual/time.texi
+++ b/manual/time.texi
@@ -503,7 +503,8 @@ Systems may support more than just these two clocks.
 
 @deftypefun int clock_gettime (clockid_t @var{clock}, struct timespec *@var{ts})
 @standards{POSIX.1, time.h}
-Get the current time accoding to the clock identified by @var{clock},
+@safety{@prelim{}@mtsafe{}@assafe{}@acsafe{}}
+Get the current time according to the clock identified by @var{clock},
 storing it as seconds and nanoseconds in @code{*@var{ts}}.
 @xref{Time Types}, for a description of @code{struct timespec}.
 
@@ -524,6 +525,7 @@ clock:
 
 @deftypefun int clock_getres (clockid_t @var{clock}, struct timespec *@var{res})
 @standards{POSIX.1, time.h}
+@safety{@prelim{}@mtsafe{}@assafe{}@acsafe{}}
 Get the actual resolution of the clock identified by @var{clock},
 storing it in @code{*@var{ts}}.
 
